File: 217/solution.go
func containsDuplicate(nums []int) bool {
	m := make(map[int]bool)
	
	for _, val := range(nums) {
			_, ok := m[val]
			if !ok {
					m[val] = true
			} else {
					return true
			}
	}
	
	return false
}

File: 217/solution.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_set<int> hashMap;
        for (int val:nums) {
            auto iter = hashMap.find(val);
            if (iter != hashMap.end()) {
                return true;
            } else {
                hashMap.insert(val);
            }
        }
        return false;
    }
};